tests/mdx/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -276,7 +276,8 @@ printWidth: 80
 
 <Hello>
   test <World /> test
-</Hello>123
+</Hello>
```

Comment:
Oh. So far it's the only incompatibility between MDX's block-level JSX and React. Before trying to fix this, I'd first make sure it's not a bug.

Reply:
Yep, another evidence proving `mdx` is not simply `jsx`, right?

Reply:
The spec has clear distinction between block-level and inline JSX. And so far it looked like the block-level one is supposed to be 100% React-compatible so that it was possible to copy-paste React code from JS to MDX. Could the creators of MDX deliberately break this use case by introducing this incompatibility? I'm not sure. We need to ask them.

Reply:
cc @johno @wooorm Any advices here?

Reply:
Also asked here https://spectrum.chat/mdx/general/whitespace-in-block-level-jsx~38817f54-db82-4971-be9e-584e1c4d20ff

Reply:
@JounQin What would you say to closing this to merging this PR as is and opening an issue about that whitespace?

Reply:
Cool to me then, but we should post an issue with reference first before merging.

Reply:
https://github.com/prettier/prettier/issues/6961